recipes-support/squeekboard/sssqueekboard_git.bb

```diff
@@ -0,0 +1,62 @@
+SUMMARY = "A Wayland virtual keyboard"
```

Comment:
can you remove the squeekboard extra recipes that are not required, lets not commit wip recipes